CONTRIBUTING.md

```diff
@@ -224,6 +225,54 @@ GitHub has a [nice introduction](https://help.github.com/articles/github-flow/)
 <img align="center" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>
 
 
+## Updating the schema
+
+Portions of the BIDS specification are defined using YAML files, in order to make the specification machine-readable.
+Currently, the only portion of the specification that relies on this schema is the Entity Table, but any changes to the specification should be mirrored in the schema.
+
+### The format of the schema
+
+The schema reflects the files and objects in the specification, as well as associations between these objects.
+Here is a list of the files and subfolders of the schema, with a description for each:
+
+- `associated_data.yaml`:
+    Folders that contain files falling outside the jurisdiction of the specification, but which are commonly contained within the same folder as a BIDS dataset.
+    The schema specifies which folders are accepted and whether they are required or optional.
+- `top_level_files.yaml`:
+    Modality-agnostic files stored at the top level of a BIDS dataset.
+    The schema specifies whether these files are required or optional, as well as acceptable extensions for each.
+- `modalities.yaml`:
+    Modalities supported by the specification, along with a list of associated datatypes.
+- `entities.yaml`:
+    A list of entities (key/value pairs in folder and filenames) with associated descriptions and formatting rules.
+    The order of the entities in the file determines the order in which entities must appear in filenames.
+- `datatypes/*.yaml`:
+    Datatypes supported by the specification.
+    Each datatype may support many suffixes.
+    These suffixes are divided into groups based on what extensions and entities are allowed for each.
+- `auxdatatypes/*.yaml`:
+    Auxiliary (not directly imaging or data-containing) datatypes supported by the specification.
+    Each auxiliary datatype is associated with a set of datatypes, and these auxiliary datatypes are grouped based on what datatypes, extensions, and entities are allowed for each.
+
+### Making a change to the schema
+
+#### 1. Ensure that changes to the specification are matched in the schema
+
+The schema formalizes the rules described in the specification text, so you must ensure that any changes which impact the rules of the specification (including, but not limited to, adding new entities, suffixes, datatypes, modalities, etc.) are reflected in the schema as well.
+
+#### 2. Generate an updated entity table before pushing your changes
+
+Run the Python script `tools/bids_schema.py`:
+
+```bash
+cd tools/
+python bids_schema.py entity ../src/schema/ ../src/99-appendices/04-entity-table.md
```

Comment:
FYI, just can confirm that I tried running this command on my Mac... had to get the right version of Python and install a few packages, but then it ran no problem.  The created MD table looks fine.

Reply:
FWIW, I see such need for manual run if the tool as temporary. Filed https://github.com/bids-standard/bids-specification/issues/544 to automate, and will look into it unless someone beats me to it